docs/api/metadata.md

```diff
@@ -0,0 +1,219 @@
+---
+id: metadata
+title: Extending Models Using Metadata
+---
+
+Saleor gives you the possibility to customize your shop without the need of any changes in the code. Once you get your own Saleor instance, you can extend most of Saleor's database models using metadata. 
+
+For example, you can add specific fields to metadata of objects to store extra information on that object. Using metadata you can also customize the way your storefront appears to your customers. 
+
+The following models contain metafields which data can be modified:
+
+* `collection`
```

Comment:
I would use `PascalCase` for model names, that is the convention in Django and in GraphQL when referring to objects.